torchdynamo/allowed_functions.py

```diff
@@ -100,6 +102,12 @@ def _allowed_function_ids():
     torch.distributions.Distribution.set_default_validate_args(False)
     torch_object_ids = dict()
 
+    def _is_allowed_module_prefix(obj):
```

Comment:
I think this will match stuff like `torchvision` or `mathfoo` which could lead to false positives.

Reply:
Thanks.

It was earlier missing objects like `torch.set_grad_enabled`. So, I kept `torch` instead of `torch.`.

I updated the check to be more restrictive but still allow `torch` objects.